stl/inc/functional

```diff
@@ -28,6 +28,12 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma push_macro("new")
 #undef new
 
+_EXTERN_C
+void __stdcall __std_function_noop_move(uintptr_t*, uintptr_t*) noexcept;
+void __stdcall __std_function_noop_destroy(uintptr_t*) noexcept;
+void __stdcall __std_function_move_large(uintptr_t* _Self, uintptr_t* _Src) noexcept;

```

Comment:
Why can't these be inline?

Reply:
They will be always called by pointer. So will never actually be inlined.
So I prefer to have them non-inline to reduce the object sizes and linker job.

Reply:
Are  you sure the compiler wouldn't be able to "devirtualize" some of the indirect calls?